stubs/parsimonious/parsimonious/nodes.pyi

```diff
@@ -27,7 +28,7 @@ class RuleDecoratorMeta(type): ...
 
 class NodeVisitor(metaclass=RuleDecoratorMeta):
     grammar: Grammar | Any
-    unwrapped_exceptions: tuple[type[Exception], ...]
+    unwrapped_exceptions: tuple[type[BaseException], ...]
```

Comment:
What was the motivation for changing this?

Reply:
https://github.com/erikrose/parsimonious/pull/221 submitted as to address where I expect your motivation is coming from.

Reply:
> Do you have a motivation for retaining it beyond it's what was already there?

No, not at all. I just couldn't see any clear evidence either way in the source code, and was curious if you'd seen something I hadn't, since you were the one proposing the change :)

Reply:
I think I agree that it makes sense to use `BaseException` here, and the change seems unlikely to cause any regressions, so let's go with it.